File: Algo/hr_implement/pickingNumberLessThanOne.cs
using System;
using System.Linq;

public class Test
{
    public void TestFun()
    {
        int[] a = {1, 2, 2, 3, 1, 2};
        int max_count=0;

        foreach(var element in a)
        {
            int count_curr =  a.Where(x=> x==element).Count();
            int count_curr_minus_one = a.Where(x=> x==element-1).Count();
            int sum_of_count = count_curr + count_curr_minus_one;

            if(sum_of_count > max_count)
            {
                max_count = sum_of_count;
            }
        }
        Console.WriteLine(max_count);
    }
}

File: Algo/hr_implement/2/pickingNumberLessThanOne.js
function pickingNumbers(a) {
    let max = 0;

    a.forEach(element => {
        let count_curr = a.filter(x=>x==element).length,
            count_curr_minus_one = a.filter(x=> x==element-1).length,
            sum_of_counts = count_curr + count_curr_minus_one;

        if(sum_of_counts > max) {
            max = sum_of_counts;
        }
    });

    return max;
}

let a = [1, 2, 2, 3, 1, 2];

console.log(pickingNumbers(a));
